python/tests/test_fixture.py

```diff
@@ -61,28 +68,20 @@ def test_up(self):
 
     def test_cli_bootstrap(self):
         """
-        The command line performs a fake bootstrap and populates JUJU_DATA.
+        The fake-juju command line tool can perform a fake bootstrap. After the
+        bootstrap the controller machine (machine 0) is present and started.
         """
         juju_data = self.useFixture(TempDir())
         self.useFixture(EnvironmentVariable("JUJU_DATA", juju_data.path))
+        self.useFixture(EnvironmentVariable("FAKE_JUJUD_CERTS", CERT))
         check_call([FAKE_JUJU, "bootstrap", "foo", "bar"])
+        check_call([FAKE_JUJU, "switch", "bar"], stdout=PIPE, stderr=STDOUT)
+
+        output = check_output([FAKE_JUJU, "status", "--format=json"])
+        status = json.loads(output)
+
+        self.assertEqual(
+            "pending", status["machines"]["0"]["machine-status"]["current"])
```

Comment:
In the PR description and the test docstring you say that the machine is "started". Why do you ensure that it's "pending" here?